ack/bin/cli.py

```diff
@@ -32,4 +33,10 @@ def cli():
         if inspect.isclass(step)
     }
 
-    fire.Fire({**step_map, "all": All})
+    # Interrupt fire print return
+    with mock.patch("fire.core._PrintResult"):
```

Comment:
Using a mock to modify some library behavior seems really hacky. Can you elaborate on what's going on here?

Reply:
`fire.Fire` attempts to pretty print the result of whatever function is called. There are some object types (namely pandas dataframes) that it doesn't know how to print and so it throws and error insteand. Because we don't care about the result of a function when called by CLI this literally interrupts and stops `fire` from printing anything instead of the potential error throw.